app/grandchallenge/evaluation/models.py

```diff
@@ -743,6 +778,37 @@ def _clean_external_evaluation(self):
                     "An external evaluation phase must have a parent phase."
                 )
 
+    def _clean_evaluation_requirements(self):
```

Comment:
This would need a bit of rework if you take on the review suggestions but it looks good!